src/styles/block/container.scss

```diff
@@ -0,0 +1,6 @@
+.container {
+
+  &__calendar {
+    margin: 10px auto;
+  }
+}
```

Comment:
Add an empty line in the end of the file

Reply:
**Fixed**